### PR TITLE
Add Gradle Play Publisher

### LINKED ISSUE #9074: Gradle Play Publisher (GPP) by Triple-T
### Brand Name

Gradle Play Publisher (GPP) by Triple-T

### Website

https://github.com/Triple-T/gradle-play-publisher

### Popularity Metric

Although Gradle Play Publisher (GPP) by Triple-T does not have more than 5k stars yet I suppose it is a widely used plugin.

### Official Resources for Icon and Color

gpp

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by CrazyMarvin**
**Issue:** closes #9074

**Similarweb rank:**
  

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description

The [Gradle Play Publisher](https://github.com/Triple-T/gradle-play-publisher) from @Triple-T is the most used and famous  release automation tool next to [fastlane](https://fastlane.tools/).

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5215,6 +5215,21 @@
...
+            "source": "https://github.com/Triple-T/gradle-play-publisher/blob/master/assets/logo.svg",
+            "license": {
+                "type": "MIT",
+                "url": "https://github.com/Triple-T/gradle-play-publisher/blob/master/LICENSE"
+            },
+            "aliases": {
+                "aka": [
+                    "gpp",
+                    "Triple-T"
+                    ]
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:5230]:
```suggestion
                ]
```
Failing at linter. This correction should fix the issue. You can run `npm run lint` before comitting to check for any issues.

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

**Comment by CrazyMarvin** [comment]
Thanks for your help! 😊

File: _data/simple-icons.json [hunk 2]
```diff
@@ -5215,6 +5215,21 @@
             "source": "https://gradle.com/brand",
             "guidelines": "https://gradle.com/brand"
         },
+        {
+            "title": "Gradle Play Publisher",
+            "hex": "82B816",
+            "source": "https://github.com/Triple-T/gradle-play-publisher/blob/master/assets/logo.svg",
+            "license": {
+                "type": "MIT",
+                "url": "https://github.com/Triple-T/gradle-play-publisher/blob/master/LICENSE"
```
[the remark this answers is not in the document]
Comment (adamrusted) [reply _data/simple-icons.json]: If it's a standard license, no. You can get away with just saying `"type": "MIT"` - only needs the URL if it's MIT with caveats, or if it's a custom license.

File: _data/simple-icons.json [hunk 3]
```diff
@@ -5225,6 +5225,21 @@
             "source": "https://gradle.com/brand",
             "guidelines": "https://gradle.com/brand"
         },
+        {
+            "title": "Gradle Play Publisher",
+            "hex": "82B816",
+            "source": "https://github.com/Triple-T/gradle-play-publisher/blob/df4eadf1ca6b5bad50e21be0b21816722ed50342/assets/logo.svg",
+            "license": {
+                "type": "MIT",
+                "url": "https://github.com/Triple-T/gradle-play-publisher/blob/df4eadf1ca6b5bad50e21be0b21816722ed50342/LICENSE"
```
Comment (adamrusted) [inline _data/simple-icons.json:5234]:
```suggestion
                "type": "MIT"
```

Assuming it's a 'vanilla' MIT license (doesn't have any caveats) then we can remove the URL here!

**Comment by CrazyMarvin** [comment]
Why is the linter failing? 😅

**Comment by adamrusted** [comment]
> Why is the linter failing? 😅

I'm looking into that right now, bear with.

File: _data/simple-icons.json [hunk 4]
```diff
@@ -5225,6 +5225,20 @@
...
+            "source": "https://github.com/Triple-T/gradle-play-publisher/blob/df4eadf1ca6b5bad50e21be0b21816722ed50342/assets/logo.svg",
+            "license": {
+                "type": "MIT"
+            },
+            "aliases": {
+                "aka": [
+                    "gpp",
+                    "Triple-T"
+                ]
+            }
```
Comment (adamrusted) [inline _data/simple-icons.json:5240]: Believe it or not, it's this that is causing the linter issue...

```suggestion
            }
```

Comment (PratyushJayachandran) [reply _data/simple-icons.json:5240]: Weirdly this issue was not showing on codespaces. I only did the change that I requested on the PR and the linter was passing.
Anyway `npm run add-icon-data` never fails

Comment (CrazyMarvin) [reply _data/simple-icons.json]: You are awesome! 👍

**Review by adamrusted — APPROVED** [review]
Had a quick check of the logo while we were having linter errors earlier - perfect match.
Thanks for your work (and patience) on this @CrazyMarvin and also to @PratyushJayachandran for your support!

**Comment by adamrusted** [comment]
@CrazyMarvin - it would appear I don't have access to update your branch, could you please merge with develop and I'll get this merged in.

**Pull request merged**